WcaOnRails/app/models/competition.rb

```diff
@@ -1542,6 +1556,7 @@ def set_wcif_events!(wcif_events, current_user)
           raise WcaExceptions::BadApiParameter.new("Cannot update events")
         end
         competition_events.find_by_event_id!(wcif_event["id"]).load_wcif!(wcif_event)
+        competition_events.find_by_event_id!(wcif_event["id"]).save
```

Comment:
Why is this becoming necessary now? The method is pretty large so probably I'm overlooking something really obvious, but I don't see what this `save` invocation has to do with qualification limits.